### ISSUE TITLE
Add issue templates

### ISSUE THREAD

**Issue by PgBiel** [typst/hayagriva#201]
Should help people use the appropriate labels and formats for issues.

**Comment by laurmaedje** [comment]
We could also add a link to Typst's issue choice picker to encourage people to report bibliography bugs here directly. Otherwise the templates will often not be used.

**Comment by PgBiel** [comment]
Good idea , though I think both ideas complement each other (the hayagriva link would lead to an issue template made for bib bugs, which adds the bug label and stuff)
